lib/expand/join.go

```diff
@@ -98,6 +101,8 @@ type Peer struct {
 	// server is the gRPC installer server
 	server *server.Server
 
+	// abortC is a channel to signal external operation abort
+	abortC     chan chan struct{}
```

Comment:
Nit: It's not clear, atleast based on the comment, why this is a `chan chan struct{}` as opposed to a `chan struct{}` like a stopC that's a sort of common go pattern?